12-designing-ux-for-ai-applications/README.md

```diff
@@ -1,120 +1,121 @@
-# Designing UX for AI Applications
```

Comment:
@yoshioterada It looks like you've translated section 12. directly from the original text. Is this correct?